README.md

```diff
@@ -203,67 +186,27 @@ You may also want to keep all custom serializers you've defined.
 
 ### Multiplatform (common, JS, Native)
 
-Platform artifacts have the same names as JVM one, but with additional suffix (e.g. `org.jetbrains.kotlinx:kotlinx-serialization-runtime-native`). For Native artifact, Gradle metadata is required (put the line `enableFeaturePreview('GRADLE_METADATA')` in your `gradle.properties`) and minimal supported version of Gradle is 5.3.
-
-Typically, you need the following dependencies in your multiplatform project (don't forget to rename [source sets](https://kotlinlang.org/docs/reference/building-mpp-with-gradle.html#configuring-source-sets) according to your setup):
-
+Most of the modules are also available for Kotlin/JS and Kotlin/Native.
+You can add dependency to the required module right to the common source set:
 ```gradle
-sourceSets {
-    commonMain {
-        dependencies {
-            implementation kotlin('stdlib-common')
-            implementation "org.jetbrains.kotlinx:kotlinx-serialization-runtime-common:$serialization_version"
-        }
-    }
-    commonTest {
-        dependencies {
-            implementation kotlin('test-common')
-            implementation kotlin('test-annotations-common')
-        }
-    }
-    jvmMain {
-        dependencies {
-            implementation kotlin('stdlib-jdk8')
-            implementation "org.jetbrains.kotlinx:kotlinx-serialization-runtime:$serialization_version"
-        }
-    }
-    jvmTest {
-        dependencies {
-            implementation kotlin('test')
-            implementation kotlin('test-junit')
-        }
-    }
-    jsMain {
-        dependencies {
-            implementation kotlin('stdlib-js')
-            implementation "org.jetbrains.kotlinx:kotlinx-serialization-runtime-js:$serialization_version"
-        }
-    }
-    jsTest {
-        dependencies {
-            implementation kotlin('test-js')
-        }
-    }
-    nativeMain {
-        dependencies {
-            implementation "org.jetbrains.kotlinx:kotlinx-serialization-runtime-native:$serialization_version"
-        }
+commonMain {
+    dependencies {
+        implementation "org.jetbrains.kotlinx:kotlinx-serialization-core:$serialization_version"
+        implementation "org.jetbrains.kotlinx:kotlinx-serialization-protobug:$serialization_version"
```

Comment:
Typo
```suggestion
        implementation "org.jetbrains.kotlinx:kotlinx-serialization-protobuf:$serialization_version"
```

Reply:
@qwwdfsad Why do we need protobuf at all ion core readme?